### PR TITLE
Add two further piratical phrases to the dictionary

### PR DESCRIPTION
**Pull request by calhutchison**
Please check the syntax of the additional phrases I have added to the dictionary.

### CODE REVIEW

File: arrr.py [hunk 1]
```diff
@@ -81,6 +81,7 @@
     "fail": "scupper",
     "quickly": "smartly",
     "captain": "cap'n",
+    "meeting": "parley with rum and cap'n",
```
Comment (ladyrassilon) [inline arrr.py:84]: LGTM

File: arrr.py [hunk 2]
```diff
@@ -102,6 +103,7 @@
     "blisterin' barnacles!",
     "ye flounderin' nincompoop.",
     "thundering typhoons!",
+    "sling yer hook"
```
Comment (ladyrassilon) [inline arrr.py]: Missing comma at the end.

Not strictly needed, but its best practice to end a list with the last item having a comma.

**Review by ladyrassilon — CHANGES_REQUESTED** [review]
Please add the missing comma.

File: arrr.py (hunk 2 above)
Comment (calhutchison) [reply arrr.py]: This should now be correct

**Review by ladyrassilon — APPROVED** [review]
Looks Good from this end. Nice job @calhutchison

File: arrr.py (hunk 2 above)
Comment (ntoll) [reply arrr.py]: :+1:

**Pull request merged**